### PR TITLE
Add telnet

### PR DESCRIPTION
**Pull request by otherroger**
I hacked in a change to the action, am testing it with:

https://github.com/solvaholic/scaling-succotash/blob/7175593d295a7cb9ad5fb74b610602ff86407ce7/.github/workflows/octodns-sync.yml

### CODE REVIEW

**Comment by otherroger** [comment]
Without changing that workflow file I'll push another change to the PR head branch to make the workflow run again.

The comment was like this:

image

File: scaling-succotash.yaml [hunk 1]
```diff
@@ -8,6 +8,9 @@ localhost:
 server:
   type: A
   value: 10.0.0.1
+telnet:
```
Comment (solvaholic) [inline scaling-succotash.yaml]: Telnet is bad mmkay, use SSH

```suggestion
ssh:
```

**Review by solvaholic — CHANGES_REQUESTED** [review]

**Comment by otherroger** [comment]
Well. That's confusing.

**Review by solvaholic — APPROVED** [review]

**Comment by solvaholic** [comment]
Ok. I [made some change to the workflow file](https://github.com/solvaholic/scaling-succotash/commit/eb12a9897e5495c75fdeaa7f77f0dd965b31a7cb) which should get closer to what I thought would happen here.

**Comment by otherroger** [comment]
Woohoo!

image

**Pull request closed without merging**